utils/node_local_test.py

```diff
@@ -1995,6 +2010,101 @@ def test_two_mounts(self):
         if dfuse1.stop():
             self.fatal_errors = True
 
+    def test_cache_expire(self):
+        """Check that data and readdir cache expire correctly
+
+        Crete two mount points on the same container, one for testing, the second for
+        oob-modifications.
+
+        Populate directory, read files in it (simulating ls -l).
+
+        oob remove some files, create some more and write to others.
+
+        re-read directory contents, this should appear unchanged.
+
+        Wait for expiry time to pass
+
+        re-read directory contents again, now this should be up to date.
+        """
+        cache_time = 20
+
+        cont_attrs = {}
+        cont_attrs['dfuse-data-cache'] = False
+        cont_attrs['dfuse-attr-time'] = cache_time
+        cont_attrs['dfuse-dentry-time'] = cache_time
+        cont_attrs['dfuse-ndentry-time'] = cache_time
```

Comment:
This is a cut-and-paste change, yes it would better like that.  Historically it used OrderedDict() to be deterministic however that's no longer required and it didn't get updated full when that was removed.